rsmetrics.py

```diff
@@ -219,14 +231,20 @@ def inner():
     "Provider",
     "Ingestion",
 ]
+
 logging.info("Reading services...")
 run.services = pd.DataFrame(
-    list(
-        rsmetrics_db["resources"].find(
-            {"provider": {"$in": [args.provider]}}, {"_id": 0}
-        )
-    )
+    list(rsmetrics_db["resources"].find({
```

Comment:
In RS Metrics the current algorithm for choosing which users (or services) to select for the metrics calculations is:


- the user (or service) must belong to Cyfronet or Athena provider's dataset
- if the created_on and the deleted_on are not set (=None), use that user (or service)
- if the created_on is less than or equal to the endtime of the desired time range, use that user (or service)
- if the deleted_on is greater than or equal to the starttime of the desired time range, use that user (or service)